instrumentation-api/src/main/java/io/opentelemetry/instrumentation/api/tracer/BaseTracer.java

```diff
@@ -253,6 +250,10 @@ public void addThrowable(Span span, Throwable throwable) {
     return propagators.getTextMapPropagator().extract(Context.root(), carrier, getter);
   }
 
+  public <C> void inject(Context context, C carrier, TextMapSetter<C> setter) {
```

Comment:
public method in a public API should have some javadoc, I think.

Reply:
Definitely. I'm planning to make a separate PR just to add Javadocs to `BaseTracer` sometime in the near future.